### PR TITLE
Quote variables in cli_bat_text install section

### PR DESCRIPTION
**Pull request by vorce**
This allows the release batch script's `install` command to be run in a path that includes spaces.

Background: We've encountered some problems when running the batch script to install the project as a windows service and the full path to the script contains spaces. Example: `C:\Program Files\myservice\_build\prod\rel\demo\bin`

We have tested this by creating a release and running `install` on a fresh elixir project. But not sure how to add an automated test for this, any ideas?

### CODE REVIEW

**Comment by josevalim** [comment]
@vorce unfortunately this is not trivial at all to test. Have spent about 2 or 3 days getting it automated, but not dice, especially as it requires admin permission in most systems.

File: lib/mix/lib/mix/tasks/release.init.ex [hunk 1]
```diff
@@ -416,9 +416,9 @@ defmodule Mix.Tasks.Release.Init do
       goto end
     )
 
-    !ERLSRV! add !RELEASE_NAME!_!RELEASE_NAME! ^
-      -!RELEASE_DISTRIBUTION! "!RELEASE_NODE!" ^
-      -env RELEASE_ROOT=!RELEASE_ROOT! -env RELEASE_NAME=!RELEASE_NAME! -env RELEASE_VSN=!RELEASE_VSN! -env RELEASE_COOKIE=!RELEASE_COOKIE! -env RELEASE_NODE=!RELEASE_NODE! -env RELEASE_VM_ARGS=!RELEASE_VM_ARGS! -env RELEASE_TMP=!RELEASE_TMP! -env RELEASE_SYS_CONFIG=!RELEASE_SYS_CONFIG! ^
+    "!ERLSRV!" add "!RELEASE_NAME!_!RELEASE_NAME!" ^
+      -"!RELEASE_DISTRIBUTION!" "!RELEASE_NODE!" ^
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/release.init.ex]:
```suggestion
      -!RELEASE_DISTRIBUTION! "!RELEASE_NODE!" ^
```

File: lib/mix/lib/mix/tasks/release.init.ex [hunk 2]
```diff
@@ -416,9 +416,9 @@ defmodule Mix.Tasks.Release.Init do
       goto end
     )
 
-    !ERLSRV! add !RELEASE_NAME!_!RELEASE_NAME! ^
-      -!RELEASE_DISTRIBUTION! "!RELEASE_NODE!" ^
-      -env RELEASE_ROOT=!RELEASE_ROOT! -env RELEASE_NAME=!RELEASE_NAME! -env RELEASE_VSN=!RELEASE_VSN! -env RELEASE_COOKIE=!RELEASE_COOKIE! -env RELEASE_NODE=!RELEASE_NODE! -env RELEASE_VM_ARGS=!RELEASE_VM_ARGS! -env RELEASE_TMP=!RELEASE_TMP! -env RELEASE_SYS_CONFIG=!RELEASE_SYS_CONFIG! ^
+    "!ERLSRV!" add "!RELEASE_NAME!_!RELEASE_NAME!" ^
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/release.init.ex]:
```suggestion
    "!ERLSRV!" add !RELEASE_NAME!_!RELEASE_NAME! ^
```

File: lib/mix/lib/mix/tasks/release.init.ex (hunk 1 above)
Comment (josevalim) [inline lib/mix/lib/mix/tasks/release.init.ex]:
```suggestion
      -!RELEASE_DISTRIBUTION! !RELEASE_NODE! ^
```

**Comment by josevalim** [comment]
@vorce I have removed the quote in three places, where the characters are guaranteed to not have spaces or foreign characters. Can you please try them locally and let me know if it works? If they do, you can accept them and then I will merge this PR. Thank you.

**Comment by josevalim** [comment]
Actually ignore me. If they worked before without the quotes and they work with the quotes, any way works. Thanks. :)

File: lib/mix/lib/mix/tasks/release.init.ex (hunk 1 above)
Comment (josevalim) [inline lib/mix/lib/mix/tasks/release.init.ex]:
```suggestion
      -!RELEASE_DISTRIBUTION! "!RELEASE_NODE!" ^
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by vorce** [comment]
@josevalim wow that was quick, didn't even have time to answer you :)

Thanks.

**Pull request merged**